aviary/variable_info/variables.py

```diff
@@ -730,3 +730,9 @@ class Takeoff:
     class Taxi:
         DURATION = 'mission:taxi:duration'
         MACH = 'mission:taxi:mach'
+
+
+class Settings:
+    EQUATIONS_OF_MOTION = 'settings:equations_of_motion'
+    MASS_METHOD = 'settings:mass_method'
+    VERBOSITY = 'settings:verbosity'
```

Comment:
This does break the rule that we had established for Aviary-core code that there would always be three tiers in the hierarchy. I don't see the need to keep this rule and bend ourselves around for it, but I wanted to raise awareness that we are essentially doing away with it. Was that intended?

Reply:
I couldn't think of a useful "middle" tier name so I didn't include one just to have it. I'd be quite happy to remove the mission tier of dynamic at some point too, it's kind of useless.